flow/workflows/normalize_flow.go

```diff
@@ -31,18 +31,20 @@ func NewNormalizeFlowExecution(ctx workflow.Context, state *NormalizeFlowState)
 
 func NormalizeFlowWorkflow(ctx workflow.Context,
 	config *protos.FlowConnectionConfigs,
+	options *protos.NormalizeFlowOptions,
```

Comment:
normalize changes should be held off for #893

Reply:
needed to pass `TableNameSchemaMapping` in, was previously contained in config